prefab_cloud_python/_structlog_processors.py

```diff
@@ -28,46 +36,50 @@
 
 
 def clean_event_dict(_, __, event_dict):
-    event_dict.pop("pathname")
-    event_dict.pop("func_name")
-    event_dict.pop("config_client")
-    event_dict.pop("log_prefix")
-    event_dict.pop("log_boundary")
-    event_dict.pop("log_path_aggregator")
-    if "skip_aggregator" in event_dict:
-        event_dict.pop("skip_aggregator")
-    if "internal_path" in event_dict:
-        event_dict.pop("internal_path")
+    event_dict.pop(CallsiteParameter.PATHNAME.value)
+    event_dict.pop(CallsiteParameter.FUNC_NAME.value)
+    event_dict.pop(STRUCTLOG_EVENT_DICT_KEY_PREFAB_CONFIG_CLIENT)
+    event_dict.pop(STRUCTLOG_EVENT_DICT_KEY_LOG_PREFIX)
+    event_dict.pop(STRUCTLOG_EVENT_DICT_KEY_LOG_BOUNDARY)
+    event_dict.pop(STRUCTLOG_EVENT_DICT_KEY_PATH_AGGREGATOR)
+    if STRUCTLOG_EVENT_DICT_KEY_SKIP_AGGREGATOR in event_dict:
+        event_dict.pop(STRUCTLOG_EVENT_DICT_KEY_SKIP_AGGREGATOR)
+    if STRUCTLOG_EVENT_DICT_KEY_INTERNAL_PATH in event_dict:
+        event_dict.pop(STRUCTLOG_EVENT_DICT_KEY_INTERNAL_PATH)
     return event_dict
 
 
 def set_location(_, __, event_dict):
-    if "internal_path" in event_dict:
-        if event_dict["internal_path"]:
-            event_dict["location"] = (
-                "cloud.prefab.client.python.%s" % event_dict["internal_path"]
+    if STRUCTLOG_EVENT_DICT_KEY_INTERNAL_PATH in event_dict:
+        if event_dict[STRUCTLOG_EVENT_DICT_KEY_INTERNAL_PATH]:
```

Comment:
`if event_dict.get(STRUCTLOG_EVENT_DICT_KEY_INTERNAL_PATH):`

Reply:
i realized this is using STRUCTLOG_EVENT_DICT_KEY_INTERNAL_PATH presence, even set to None as an indication that the logging is internal. Added comments. May be cleaner to have an explicit flag but will let it ride for now